index.html

```diff
@@ -172,7 +172,6 @@ <h2 class="project-title">Facebook 360</h2>
     </div>
 
     <div class="project-card">
```

Comment:
well done on implementing some of the changes requested by the previous reviewer , but you still have one small thing to fix as it was requested before kindly use the same background color for this section as in figma template 

[ OPTIONAL ] also kindly do your best to replicate figma design , adjust card size , horizontal and vertical spacing to make your work section look perfect 👌

![card spacing student](https://user-images.githubusercontent.com/59707859/221207505-5f441a97-46e5-4c6a-aabe-6347985ba462.PNG)

**figma :**
![work section background figma](https://user-images.githubusercontent.com/59707859/221209776-c2559e9c-2358-4a0e-ad07-47de9ef9498b.PNG)

Reply:
@souad988  Hi and thanks for your time reviewing this. I will be requesting an extra review so I want to be sure of what is expected from me. I have been told this and used the exact background color used in the template since the beginning so I'm confused and don't really know what to change anymore. I'm attaching screenshots here 
![Cap1](https://user-images.githubusercontent.com/119133987/221222700-8de045e8-8371-41d8-b8fd-b986532ffac2.JPG)
![Cap2](https://user-images.githubusercontent.com/119133987/221222707-8c65715f-c195-42d0-abc7-8bc8db14b7b0.JPG)

Reply:
@souad988  Please sorry to bother but land me your time and look into this before I submit it again because I'm already using my extra reviews count